completions/perl

```diff
@@ -15,6 +15,15 @@ _perlfunctions()
         -- "$cur" ) )
 }
 
+_perldocs()
+{
+    printf "%s\t%s\t%s\n" "$(date)" "_perldocs" "$cur" >> /tmp/perl-helper.log
```

Comment:
This looks like a security hole: allows appending to arbitrary files to which the completion user has write access through symlinks. But the specifics here don't matter as writing to files is something we don't do in completions in the first place, see CONTRIBUTING.md. So that stuff needs to go.